module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>Try this: it uses matplotlib's object oriented interface and reduces the long variable names.</p><p> </p><pre class="ql-syntax" spellcheck="false">import freud
import matplotlib.pyplot as plt
 
fig, ax = plt.subplots()  # this might need arguments like fig, ax = plt.subplots(1, 1, figsize=(5, 3), dpi=200)
 
sf = freud.diffraction.StaticStructureFactorDirect(bins=200, k_max=25, k_min=1)
fcc_system = freud.data.UnitCell.fcc().generate_system(10, sigma_noise=0.10)
sf.compute(fcc_system)
sf.plot(ax=ax)
 
sc_system = freud.data.UnitCell.sc().generate_system(10, sigma_noise=0.10)
sf.compute(sc_system)
sf.plot(ax=ax)
 
plt.show()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>